### DISCUSSION TITLE
global hotkeys only for mediakeys - it is very sad restriction

### DISCUSSION THREAD

**Discussion by lpslp** [smplayer-dev/smplayer#556] [Ideas]
very sad that there is no options for global hotkeys for hotkey which i manually can choose. Only for mediakeys. This function are in other players (vlc, potplayer even AIMP) and because of that i think it is possible here too. 
on my keyboard (and i think on many other people's keyboards) there is no media keys AT ALL! and all we can use global hotkeys with smplayer

smplayer is very nice but without global hokeys i cant use it.

But still - thanks for great work and player!

**Comment by LandisTwo** [comment]
hmm, everything SMPlayer will do can be assigned a 'hotkey' or shortcut in:
Options > Preferences > Keyboard and mouse > Keyboard
Everything from play next to volume up/down.
There are so many options, I could never remember them all if assigned. : )
Your 'global' volume will work if your 'mixer', audio processor eg kmix alsa, pulse understands so that volume is adjusted when smp is Not in focus.
Landis.
ps.. what is really needed is, single-click > play in playlist, back!
- or a way to edit the function myself. : (

**Comment by lpslp** [comment]
i **did not say** that in SMPlayer we can't assign hotkeys - i said - **we can't use ANY hotkeys AS GLOBAL** hotkeys...please all - **read what writed**
